kotlin-analysis-api/src/main/kotlin/com/google/devtools/ksp/impl/symbol/kotlin/KSFunctionDeclarationImpl.kt

```diff
@@ -130,15 +133,6 @@ class KSFunctionDeclarationImpl(private val ktFunctionSymbol: KtFunctionLikeSymb
         TODO("Not yet implemented")
     }
 
-    override val declarations: Sequence<KSDeclaration>
-        get() = TODO("Not yet implemented")
-}
-
-@OptIn(InvalidWayOfUsingAnalysisSession::class)
-internal inline fun <R> analyzeWithSymbolAsContext(
-    contextSymbol: KtSymbol,
-    action: KtAnalysisSession.() -> R
-): R {
-    return KtAnalysisSessionProvider
-        .getInstance(contextSymbol.psi!!.project).analyzeWithSymbolAsContext(contextSymbol, action)
+    // TODO: Implement with PSI
```

Comment:
If this is blocking the test case, it's better to modify or disable the test instead of making it return a incorrect result.

Reply:
It's kinda tricky, the fix will be quite heavy since it will require analyzing of Psi structures, and at the same time top down visitor that visits declarations are widely used across all tests, even though majority of the tests do not care about function local members at all, therefore I wanted to put a TODO comment here to unblock other development works.

Reply:
SGTM